nni/tools/nnictl/config_schema.py

```diff
@@ -73,16 +69,13 @@ def __init__(self, algo_type):
         }
         self.builtin_name_schema = {}
         for k, n in self.builtin_keys.items():
-            self.builtin_name_schema[k] = {Optional(n): setChoice(n, *get_all_builtin_names(k+'s'))}
+            self.builtin_name_schema[k] = {Optional(n): setType(n, str)}
 
         self.customized_keys = set(['codeDir', 'classFileName', 'className'])
 
     def validate_class_args(self, class_args, algo_type, builtin_name):
         if not builtin_name or not class_args:
             return
-        meta = get_registered_algo_meta(builtin_name, algo_type+'s')
-        if meta and 'acceptClassArgs' in meta and meta['acceptClassArgs'] == False:
```

Comment:
what is the meaning of `acceptClassArgs`?

Reply:
Random tuner (and maybe some other tuners) does not accept any arguments. This checks no args are passed to such tuner.
Again this is V1 only and I don't want to fix.